apps/web/components/Footer.tsx

```diff
@@ -81,7 +81,7 @@ export default function Footer() {
           <Logo />
         </Flex>
         <Text pt={6} fontSize={'sm'} textAlign={'center'}>
-          © 2022 WALDO-Vision. All rights reserved.
+          © 2022 Waldo Vision-Vision. All rights reserved.
```

Comment:
Should we refer to the actual legal entity here? Also I wonder if we can even put this statement if we don't have copyright yet...

```suggestion
          © 2022 Waldo Intelligence LLC. All rights reserved.
```